### DISCUSSION TITLE
Error in config, i3 won't start, can't edit config

### DISCUSSION THREAD

**Discussion by drewngo** [i3/i3#6254] [Q&A]
New to i3, tried to change the font in the i3 config and when rebooting it gave me an error and i3 stopped working.

i3 gives me two options on the top of the screen to view errors, or edit config, but when I go to edit config, it opens it with nano, but the file is vim encrypted.

Since i3 isn't running, mod keys aren't working so I cannot open the terminal using mod+enter, is there any alternative way or did I cook my config :(

**Comment by okraits** [comment]
It seems that you somehow messed up your config. You can try to delete the file `~/.config/i3/config`, then the i3-config-wizard will be run on the next start of i3 and you can start over with the default config.
